README.md

```diff
@@ -58,35 +58,44 @@ pip install similarity-processor
 ### Commandline
 
 ```sh
->>>python -m similarity_processor.similarity_cmd --p "path\to\TestBank.xlsx" --u 0 --c "1,2,3" --n 8
+>>>python -m similarity --p "path\to\TestBank.xlsx" --u 0 --c "1,2,3" --n 8
 ```
 
 - Help option can be found at,  
 
 ```sh
->>>python -m similarity_processor.similarity_cmd --h
+>>>python -m similarity --h
 ```
 
 ### Code
 
 ```sh
->>> from similarity_processor.similarity_io import SimilarityIO
->>> similarity_io_obj = SimilarityIO("path\to\TestBank.xlsx", 0, "1,2,3", 11, 0, None)
+>>> from similarity.similarity_io import SimilarityIO
+>>> similarity_io_obj = SimilarityIO("path\to\TestBank.xlsx", 0, "1,2,3")
 >>> similarity_io_obj.orchestrate_similarity()
 ```
 
-Arguments:
+### Arguments
 
-1. Path to the input file  
+Mandatory
+
+1. Path to the input file
```

Comment:
Nitpick comment (pls ignore if you find it not useful)

In my editing of the *.md file, sometimes I have to change the order of the arguments, then I have to always change the numbers too. Since the order doesn't matter, instead of a number, would a bullet-point help? This way, for future edits you don't have to bother about number changes if any, or also bother about always adding new arguments to the end (to avoid number changes!)